cloudwatt/test_snat_lbaas/input.tf

```diff
@@ -6,11 +6,11 @@ variable "key_pair" {}
 
 # Backend
```

Comment:
Modifications here are not related to inter-tenant stuff :thinking: 
This modification should not be tracked